internal/pkg/pct/pct_test.go

```diff
@@ -172,7 +221,31 @@ func TestDeploy(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			if got := Deploy(tt.args.info); !reflect.DeepEqual(got, tt.want) {
+			fs := afero.NewMemMapFs()
+			afs := &afero.Afero{Fs: fs}
+			iofs := &afero.IOFS{Fs: fs}
+
+			// Create the template
+			templateDir := filepath.Join(tt.args.info.TemplateCache, tt.args.info.SelectedTemplate)
+			contentDir := filepath.Join(templateDir, "content")
+			afs.MkdirAll(contentDir, 755) //nolint:errcheck
```

Comment:
Similar to my earlier comment:
```suggestion
			afs.MkdirAll(contentDir, 750)
```